packages/schema/src/dialects/postgres.ts

```diff
@@ -121,10 +121,16 @@ export default class Postgres extends KnexPostgres implements SchemaInspector {
 		}
 
 		for (const { table_name, column_name } of primaryKeys) {
-			overview[table_name].primary = column_name;
+			if (table_name in overview) {
+				overview[table_name].primary = column_name;
+			} else {
+				// TODO: maybe display an error message to the user, so postgres permissions the directus user/role get's fixed
+			}
```

Comment:
This worries me greatly. Directus expects a primary key field to exist, so by silently not-setting it here there will be a lot of other hard-to-catch bugs introduced 😬

Reply:
The problem with your current approach is that users/admin do net get told which table actually has the problematic layout. What would be the proper way to display an (breaking) error message or throw an exception giving the the user more details on what they have to fix in their schema?